openedx/core/djangoapps/user_api/accounts/views.py

```diff
@@ -230,13 +223,12 @@ def partial_update(self, request, username):
         return Response(account_settings)
 
 
+@view_auth_classes(permission_classes=(CanDeactivateUser,))
 class AccountDeactivationView(APIView):
     """
     Account deactivation viewset. Currently only supports POST requests.
     Only admins can deactivate accounts.
     """
-    authentication_classes = (JwtAuthentication, )
```

Comment:
Here is an example where we adding authentication classes, and it's unclear whether we want to.